### ISSUE TITLE
soft-DTW loss became inf when setting bandwidth bigger than 0

### ISSUE THREAD

**Issue by LuckerYi** [Maghoumi/pytorch-softdtw-cuda#8]
[2021-05-10 17:26:05,123][torchtts.hooks.logging][INFO] - epoch = 1, step = 763, soft_dtw_mel_loss = inf, total_dur_loss = 37.49, loss = inf, lr = 2.721e-06
pred_mel.shape= torch.Size([6, 386, 80])
batch[mel].shape= torch.Size([6, 302, 80])
[2021-05-10 17:26:05,962][torchtts.hooks.logging][INFO] - epoch = 1, step = 764, soft_dtw_mel_loss = inf, total_dur_loss = 45.59, loss = inf, lr = 2.724e-06
pred_mel.shape= torch.Size([24, 113, 80])
batch[mel].shape= torch.Size([24, 78, 80])
[2021-05-10 17:26:06,781][torchtts.hooks.logging][INFO] - epoch = 1, step = 765, soft_dtw_mel_loss = 134.2, total_dur_loss = 14.37, loss = 148.6, lr = 2.728e-06
pred_mel.shape= torch.Size([6, 288, 80])
batch[mel].shape= torch.Size([6, 257, 80])
[2021-05-10 17:26:07,412][torchtts.hooks.logging][INFO] - epoch = 1, step = 766, soft_dtw_mel_loss = 305.9, total_dur_loss = 12.17, loss = 318.1, lr = 2.731e-06
pred_mel.shape= torch.Size([4, 526, 80])
batch[mel].shape= torch.Size([4, 499, 80])
[2021-05-10 17:26:08,161][torchtts.hooks.logging][INFO] - epoch = 1, step = 767, soft_dtw_mel_loss = inf, total_dur_loss = 41.76, loss = inf, lr = 2.735e-06
pred_mel.shape= torch.Size([4, 532, 80])
batch[mel].shape= torch.Size([4, 495, 80])
[2021-05-10 17:26:08,953][torchtts.hooks.logging][INFO] - epoch = 1, step = 768, soft_dtw_mel_loss = 629.2, total_dur_loss = 38.73, loss = 668, lr = 2.739e-06
pred_mel.shape= torch.Size([12, 156, 80])
batch[mel].shape= torch.Size([12, 135, 80])
[2021-05-10 17:26:09,657][torchtts.hooks.logging][INFO] - epoch = 1, step = 769, soft_dtw_mel_loss = 195.1, total_dur_loss = 18.41, loss = 213.6, lr = 2.742e-06
pred_mel.shape= torch.Size([6, 343, 80])
batch[mel].shape= torch.Size([6, 311, 80])
[2021-05-10 17:26:10,349][torchtts.hooks.logging][INFO] - epoch = 1, step = 770, soft_dtw_mel_loss = inf, total_dur_loss = 36.64, loss = inf, lr = 2.746e-06
pred_mel.shape= torch.Size([6, 344, 80])
batch[mel].shape= torch.Size([6, 279, 80])
[2021-05-10 17:26:11,073][torchtts.hooks.logging][INFO] - epoch = 1, step = 771, soft_dtw_mel_loss = inf, total_dur_loss = 30.31, loss = inf, lr = 2.749e-06
pred_mel.shape= torch.Size([9, 264, 80])
batch[mel].shape= torch.Size([9, 214, 80])
[2021-05-10 17:26:11,904][torchtts.hooks.logging][INFO] - epoch = 1, step = 772, soft_dtw_mel_loss = inf, total_dur_loss = 20.1, loss = inf, lr = 2.753e-06
pred_mel.shape= torch.Size([6, 316, 80])
batch[mel].shape= torch.Size([6, 283, 80])
[2021-05-10 17:26:12,653][torchtts.hooks.logging][INFO] - epoch = 1, step = 773, soft_dtw_mel_loss = 373.4, total_dur_loss = 27.09, loss = 400.5, lr = 2.756e-06
pred_mel.shape= torch.Size([4, 428, 80])
batch[mel].shape= torch.Size([4, 399, 80])
[2021-05-10 17:26:13,337][torchtts.hooks.logging][INFO] - epoch = 1, step = 774, soft_dtw_mel_loss = 500.6, total_dur_loss = 20.63, loss = 521.2, lr = 2.760e-06

**Comment by LuckerYi** [comment]
This issue happened from time to time, especially at the beginning of the model training phase
